service/src/test/java/bio/terra/externalcreds/dataAccess/AccessTokenCacheDAOTest.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.externalcreds.dataAccess;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import bio.terra.externalcreds.BaseTest;
+import bio.terra.externalcreds.TestUtils;
+import java.util.Optional;
+import java.util.UUID;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class AccessTokenCacheDAOTest extends BaseTest {
+
+  @Autowired private LinkedAccountDAO linkedAccountDAO;
+  @Autowired private AccessTokenCacheDAO accessTokenCacheDAO;
+
+  @Test
+  void testGetMissingAccessTokenCacheEntry() {
```

Comment:
Nit: it's nice to have the AAA comments in tests, but these aren't too hard to read so not blocking